scripts/docker/borgbackup

```diff
@@ -0,0 +1,133 @@
+#!/bin/bash
+
+set -euo pipefail
+
+function echoerr {
+  cat <<< "$@" 1>&2;
+}
+
+function quit {
+  if [ -n "${SSHFS:-}" ]; then
+    fusermount -u "$BORG_REPO"
+  fi
+
+  if [ -n "${1:-}" ]; then
+    exit "$1"
+  fi
+
+  exit 0
+}
+
+if [ -n "${SSHFS:-}" ]; then
+  if [ -n "${SSHFS_IDENTITY_FILE:-}" ]; then
+    if [ ! -f "$SSHFS_IDENTITY_FILE" ] && [ -n "${SSHFS_GEN_IDENTITY_FILE:-}" ]; then
+      ssh-keygen -t rsa -b 4096 -N '' -f "$SSHFS_IDENTITY_FILE"
+      cat "${SSHFS_IDENTITY_FILE}.pub"
+      exit 0
+    fi
+    SSHFS_IDENTITY_FILE="-o IdentityFile=${SSHFS_IDENTITY_FILE}"
+  else
+    SSHFS_IDENTITY_FILE=''
+  fi
+  if [ -n "${SSHFS_PASSWORD:-}" ]; then
+    SSHFS_PASSWORD="echo ${SSHFS_PASSWORD} |"
+    SSHFS_PASSWORD_OPT='-o password_stdin'
+  else
+    SSHFS_PASSWORD=''
+    SSHFS_PASSWORD_OPT=''
+  fi
+  mkdir -p /mnt/sshfs
+  eval "${SSHFS_PASSWORD} sshfs -o StrictHostKeyChecking=no ${SSHFS} /mnt/sshfs ${SSHFS_IDENTITY_FILE} ${SSHFS_PASSWORD_OPT}"
```

Comment:
`StrictHostKeyChecking=no`? Please, by all means, do not do this. This image is likely intended to be used as a client with remote servers, so not checking the server key could allow MITM attacks…

So better pass in the host key either as a variable or just use SSH's default [TOFU](https://en.wikipedia.org/wiki/Trust_on_first_use) security scheme, at least – with allowing the user to confirm the step/check the key manually. After all, they will run this manually one time at least, so that should work.